core/common/src/thrift/file_system_worker.thrift

```diff
@@ -0,0 +1,39 @@
+namespace java alluxio.thrift
+
+include "common.thrift"
+include "exception.thrift"
+
+struct UFSCancelFileTOptions {
+}
+
+struct UFSCompleteFileTOptions {
+}
+
+struct UFSCreateFileTOptions {
+}
+
+/**
+ * This interface contains file system worker service endpoints for Alluxio clients.
+ */
+service FileSystemWorkerClientService extends common.AlluxioService {
+  /**
+   * Cancels a file which has not been completed in the under file system.
+   */
+  void ufsCancelFile( /** the worker file id of the ufs file */ 1: i64 workerFileId,
```

Comment:
how about `cancelUfsFile` rather than `ufsCancelFile`?
also `completeUfsFile` and `createUfsFile`?
I don't have strong preference, feel free to keep your current names.